test/resample.jl

```diff
@@ -0,0 +1,53 @@
+using DSP
+using Base.Test
+
+# AM Modulator
+# sig(t) = [(1 + sin(2π*0.005*t)) * sin(2π*.05*t) for t in t]
+x_ml   = readdlm(joinpath(dirname(@__FILE__), "data", "resample_x.txt"),'\t')[:,1]
+
+#
+# [y1,b1] = resample(x, 1, 2)
+#
+rate   = 1//2
+h1_ml  = readdlm(joinpath(dirname(@__FILE__), "data", "resample_taps_1_2.txt"),'\t')[:,1]
+y1_ml  = readdlm(joinpath(dirname(@__FILE__), "data", "resample_y_1_2.txt"),'\t')[:,1]
+ty1_ml = [0:length(y1_ml)-1] / float64(rate)
```

Comment:
This triggers a deprecation warning on Julia 0.4, where you need `[a;]` to convert an AbstractVector to a Vector and `float64(x)` has been changed to `Float64(x)` (actually `@compat Float64(x)` to maintain 0.3 compatibility). But it looks like these time vectors aren't used and could just be removed?